single.php

```diff
@@ -21,6 +21,16 @@
 $post            = Timber::query_post( false, 'P4_Post' );
 $context['post'] = $post;
 
+// Strip Take Action Boxout block from the post content to add outside the normal block container.
+if ( false !== strpos( $post->post_content, '[shortcake_take_action_boxout' ) ) {
+
+	$take_action_boxout_block_start = strpos( $post->post_content, '[shortcake_take_action_boxout' );
+	$take_action_boxout_block_end   = strpos( $post->post_content, ']', $take_action_boxout_block_start );
+	$take_action_boxout_block       = substr( $post->post_content, $take_action_boxout_block_start, $take_action_boxout_block_end );
```

Comment:
this won't work. substr needs length of the string as the 3rd parameter not an index.

Reply:
calculated the length and added used that instead.